db/migrate/20230719092349_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Similarly here, kindly make sure to add all the columns as suggested by the last reviewer.  Right now, you are missing a column that will create `author_id` and a corresponding `index` and a column that will create `post_id` and a corresponding `index` 📌  You can go through [this article](https://edgeguides.rubyonrails.org/active_record_migrations.html#creating-associations-using-references) to learn about it ℹ


```
  create_table :likes do |t|
    t.timestamps 
    t.references :author, foreign_key: { to_table: 'users' }
    t.references :post, foreign_key: true 
  end
```

_Kindly note that you can also add column and index in separate commands_